FibonacciSeries.java

```diff
@@ -0,0 +1,15 @@
+class FibonacciSeries{
+public static void main(String args[])
+{  
+ int n1=0,n2=1,n3,i,count=10;  
+ System.out.print(n1+" "+n2);//printing 0 and 1  
+  
+ for(i=2;i<count;++i)//loop starts from 2 because 0 and 1 are already printed  
+ {  
+  n3=n1+n2;  
+  System.out.print(" "+n3);  
+  n1=n2;  
+  n2=n3;  
+ }  
+
+}}

```

Comment:
Can you please indent the code as it will increase the readability of the code. 

Thank you for contributing.